atomics/include/desul/atomics/Compare_Exchange_MSVC.hpp

```diff
@@ -14,95 +15,187 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #ifndef DESUL_HAVE_16BYTE_COMPARE_AND_SWAP
 #define DESUL_HAVE_16BYTE_COMPARE_AND_SWAP
 #endif
-#include <windows.h>
 
 namespace desul {
+
+template<class T, class MemoryOrder, class MemoryScope>
+T atomic_exchange(T* const, T val, MemoryOrder, MemoryScope) { return val;}
```

Comment:
Per conversation with @trws, make sure we discuss this.

Reply:
Uh oh yeah this needs to use the same lock based fallback as below for atomic_compare_exchange

Reply:
Fixed